### PR TITLE
Fund functionality

### LINKED ISSUE #2: Create Button for depositing + view on amount
Create a button for depositing + add view for how much was deposited.

### PR DESCRIPTION
**Pull request by LinusWeigand**
- fund contract with specified amount
- amount funded gets displayed
- withdraw a certain amount

### CODE REVIEW

**Comment by yvesbou** [comment]
Screenshot 2022-04-19 at 15 35 43

Formatting is not correct yet

**Comment by yvesbou** [comment]
Screenshot 2022-04-19 at 15 38 16

when I refresh the page, the deposited amount is 0 again

**Comment by yvesbou** [comment]
withdrawing and depositing works :) 👍

**Comment by leovct** [comment]
Very good!
- Can you rename the `utils` folder into `abis` to follow the gmx repository guidelines?
- PR#20 (contract repository) introduces events when user funds and withdraws money. It could be used to update the value displayed to the user (amountFundedByAddress).
- We should also display the total value of the fund.

File: src/App.js [hunk 1]
```diff
@@ -54,6 +58,62 @@ export default function App() {
...
+    console.log(amountFunded.toString());
+    try {
+      const { ethereum } = window;
+      if (ethereum) {
+        const provider = new ethers.providers.Web3Provider(ethereum);
+        const signer = provider.getSigner();
+        const fundContract = new ethers.Contract(contractAddress, contractABI, signer);
+
+        setAmountFunded(BigNumber.from((await fundContract.getAddressToAmountFunded(currentAccount)).toHexString()));
+
```
Comment (yvesbou) [inline src/App.js:107]: Please replace 106 through 108 with the following :) It fixes formatting of the displayed funded avax

```
const hexNumber = await fundContract.getAddressToAmountFunded(currentAccount);
const avaxFunded = ethers.utils.formatEther(hexNumber);
setAmountFunded(avaxFunded);
```

**Review by yvesbou — APPROVED** [review]

**Pull request merged**